websites/T/Tiermaker/presence.ts

```diff
@@ -0,0 +1,92 @@
+let episode: string,
+    creator: string,
+    c_title: string,
+    c_category: string,
+    c_stateNum: number,
+    c_currentState: string;
+
+const presence = new Presence({
+    clientId: "919901175640358983"
+  }),
+
+  startStamp = Math.floor(Date.now() / 1000),
+  urlpath = document.location.pathname.split("/"),
+
+  getElement = (query: string): string | undefined => {
+    return document.querySelector(query)?.textContent;
+  },
+
+  getInformation = (query: string): string | undefined => {
+    episode = document.querySelector(query)?.textContent;
+    return episode
+  };
+
+  
+function c_GetDetails(): string | undefined {
+  c_title = (<HTMLInputElement>document.getElementById('title')).value
+  c_category = (<HTMLInputElement>document.getElementById('select')).value
+  return
+}
+
+setInterval(c_GetDetails, 15000)

```

Comment:
If you only use c_GetDetails here you can remove the function declaration and move it here instead
```suggestion
setInterval(c_GetDetails, 15_000)
```